client/src/components/EditPlace.js

```diff
@@ -90,7 +140,19 @@ export class EditPlace extends React.Component {
             </div>
           </div>
         </div>
+      );
+    }
+    return (
+      <div>
+        <h5>Loading...</h5>
       </div>
     );
+  };
+  render() {
+    return (
+      <form onSubmit={e => this.handleSubmit(e)}>
```

Comment:
You already use arrow function in the handleSubmit, you don't need to use it here again.